HexCtrl/src/CHexCtrl.cpp

```diff
@@ -3524,11 +3524,11 @@ void CHexCtrl::UpdateInfoText()
 		if (IsSectorVisible())
 		{
 			if (m_fOffsetAsHex)
-				swprintf_s(wBuff, _countof(wBuff), L"%s: 0x%llX-0x%llX; ",
-					m_wstrSectorName.data(), GetCaretPos() / m_dwSectorSize, m_ullDataSize / m_dwSectorSize);
+				swprintf_s(wBuff, _countof(wBuff), L"%s: 0x%llX/0x%llX; ",
+					m_wstrSectorName.data(), GetCaretPos() / m_dwSectorSize, (m_ullDataSize / m_dwSectorSize)-1);
 			else
 				swprintf_s(wBuff, _countof(wBuff), L"%s: %llu-%llu; ",
-					m_wstrSectorName.data(), GetCaretPos() / m_dwSectorSize, m_ullDataSize / m_dwSectorSize);
+					m_wstrSectorName.data(), GetCaretPos() / m_dwSectorSize, (m_ullDataSize / m_dwSectorSize)-1);

```

Comment:
Why would we `minus 1` here? I don't get it.
I can agree that my calculation is slightly incorrect (overlooked it). But it's a bit more complicated.
The correct one would be:
```cpp
TotalSectors = m_ullDataSize % m_dwSectorSize ? m_ullDataSize / m_dwSectorSize + 1 : m_ullDataSize / m_dwSectorSize;
```
I have now fixed that.